### ISSUE TITLE
training accuracy 3.41？

### ISSUE THREAD

**Issue by liqidong** [choyingw/SynergyNet#12]
I followed the training steps, but my final accuracy was only more than 3.8. How can I reproduce the accuracy of 3.41?

**Comment by choyingw** [comment]
We chose the best checkpoint among all epochs. You can save more checkpoints during training and evaluate them to find the best one.

**Comment by liqidong** [comment]
I tried loading the pre-trained model and saving each batch. The highest accuracy is about 3.58, but it cannot reach 3.41. At the same time, I load the best model (3.41) for iterative training, and usually the accuracy will decrease. How to solve this problem?

**Comment by vujadeyoon** [comment]
Dear all,

I am interested in the SynergeNet.
I also encountered the issue and wonder if @liqidong solved the issue.

As far as @choyingw mentioned above, I saved all checkpoints every epochs and also saved the checkpoint corresponding to best performance in terms of NME. However, I could not reproduce the result (i.e. AFLW2000-3D-All-NME: 3.41) unforuntaetly. In my case, the lowest NME is 3.628.

I understand there may be some error between orinals and mine because of random seed, although I set fixed seed.
Thus, there is two questions.

1. Is the released code in this repository orinal code corresponding to the paper, [Synergy between 3DMM and 3D Landmarks for Accurate 3D Facial Geometry](https://arxiv.org/abs/2110.09772)? 
In the paper, the experiments are conducted on four NVIDIA GTX 1080Ti, whereas the experiment may be performed with a single NVIDIA GTX 3090 in README.md in this reposiory.
Also, unfortunately the performance of the released pretrained model is not qual to that of the paper.

2. As far as I mentioned question 1, I think the uploaded checkpoints is not the checkpoint corresponding to the official paper.
Could I get the official checkpoint which can make same results as the recorded in the paper?

Best regards,
Vujadeyoon

**Comment by choyingw** [comment]
This is the official repo, and the released checkpoints correspond to the results we reported in the paper.

**Comment by starhiking** [comment]
Could you please share the training log file about the 3.41 nme.
